client/webserver/site/src/html/wallets.tmpl

```diff
@@ -147,6 +149,64 @@
       </div>
       <div class="fs15 pt-3 text-center d-hide errcolor" id="withdrawErr"></div>
     </form>
+    
+    {{- /* RECONFIGURE WALLET */ -}}
+    <form class="card bg1 pb-3 d-hide mt-3" id="walletReconfig" autocomplete="off">
+      <div class="bg2 px-2 py-1 text-center fs18">
+        Reconfigure
+        <img id="recfgAssetLogo" class="micro-icon mx-1">
+        <span id="recfgAssetName"></span>
+        Wallet
+      </div>
+      <div class="px-3 py-1" id="reconfigInputs">
+        {{template "walletConfigTemplates"}}
+      </div>
+      <hr class="dashed my-2 mx-4">
+      <div class="d-flex px-4 mt-1">
+        <div class="col-12 p-0">
+          <label for="reconfigPW" class="pl-1 mb-1">App Password</label>
+          <input type="password" class="form-control select" id="reconfigPW" autocomplete="off">
+        </div>
+        <div class="col-12 p-0 text-right">
+          <div>&nbsp;</div>
+          <button id="submitReconfig" type="button" class="w-75 mt-1 justify-content-center fs15 bg2 selected">Submit</button>
+        </div>
+      </div>
+      <div class="fs15 pt-3 text-center d-hide errcolor" id="reconfigErr"></div>
+      <hr class="dashed mt-4 mx-4">
+      <div class="pt-2 pl-1">
+        <span class="mx-4">Or, you can...</span>
+        <ul class="fs15 mt-3 mr-4">
+          <li id="changePW" class="hoverbg pointer">Change the wallet password</li>
```

Comment:
This is slightly confusing as it suggest the user can change their wallet's passphrase with the dex software, but this is really updating the dex software's understanding of the wallet's passphrase. That is, it's not calling `walletpassphrasechange`; it's just getting a new password to use when unlocking the wallet.  So maybe here "Update the wallet password".  And maybe similar wording changes with more verbiage in the `walletRepw` form?